doc/conf.py

```diff
@@ -177,6 +180,15 @@
      'Miscellaneous'),
 ]
 
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None),
+    'mne': ('https://mne.tools/dev', None),
+    'numpy': ('https://numpy.org/devdocs', None),
+    'scipy': ('https://scipy.github.io/devdocs', None),
+    'matplotlib': ('https://matplotlib.org', None)
+}
+intersphinx_timeout = 5
+
```

Comment:
So all of the external links to other projects are done with intersphinx? I'm just trying to understand better how this all works....

Reply:
Just the links to the API pages.

Reply:
e.g., the link here to plt.subplots:

![image](https://user-images.githubusercontent.com/15852194/105121079-77bc5d80-5aa1-11eb-9fd9-12eb356e53c8.png)